control/firstboot.xml

```diff
@@ -183,7 +183,7 @@
                 </module>
                 <module>
                     <label>Customer Center</label>
-                    <name>inst_suse_register</name>
+                    <name>registration</name>
                     <enabled config:type="boolean">false</enabled>
```

Comment:
Yes. In [this case](https://bugzilla.suse.com/show_bug.cgi?id=970572) the customer changed the default template quite a lot and enabled the registration step.